### ISSUE TITLE
Factuality Evaluator failing 

### ISSUE THREAD

**Issue by ishaan-jaff** [braintrustdata/autoevals#28]
Tried this code snippet:

```python
from autoevals.llm import *
import openai

openai.api_key = "sk-"
 
# Create a new LLM-based evaluator
evaluator = Factuality()
 
# Evaluate an example LLM completion
input = "Which country has the highest population?"
output = "People's Republic of China"
expected = "China"
 
result = evaluator(output, expected, input=input)
print(result)
 
# The evaluator returns a score from [0,1] and includes the raw outputs from the evaluator
print(f"Factuality score: {result.score}")
print(f"Factuality metadata: {result.metadata['rationale']}")
```

I see this error:
Score(name='Factuality', score=0, metadata={}, error=KeyError('usage'))
Factuality score: 0

```python
Traceback (most recent call last):
  File "/Users/ishaanjaffer/Github/litellm/litellm/tests/test_autoeval.py", line 19, in <module>
    print(f"Factuality metadata: {result.metadata['rationale']}")
KeyError: 'rationale'
```

Any suggestions on how i can debug this ?

**Comment by ishaan-jaff** [comment]
nvm it worked now, i have no idea what caused the bug

**Comment by ishaan-jaff** [comment]
autoevals                                  0.0.30 
openai                                     0.28.1

python 3.10

**Comment by ishaan-jaff** [comment]
it would also be useful if you print the stacktrace on errors

**Comment by ankrgyl** [comment]
Interesting, do you mind patching #27 or re-testing after we land that change? I was not able to repro the error, but I suspect the response you're getting from OpenAI (perhaps related to your key?) is missing the `usage` field.

**Comment by ishaan-jaff** [comment]
yes, i can re test once you've landed https://github.com/braintrustdata/autoevals/pull/27 , should i leave this issue open till then ?

**Comment by ankrgyl** [comment]
just published 0.0.31. Please leave it opne!

**Comment by ishaan-jaff** [comment]
works now
